doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -561,7 +561,7 @@ might be useful in understanding the format.
 The Simple Query Language provides a simplified syntax to retrieve entities which match a set of
 conditions.
 A query is composed by a list of statements separated by the ';' character.
-Each statement express a matching condition.
+Each statement expresses a matching condition.
```

Comment:
Maybe this should be rollbacked, waiting for the next proof-read process upon releasing next RC based on latest.

Maybe I bit contra-natural :) but note that if we start fixing proof-read errors now, then a lot of noise will be added in the future when we compare current RC with lastet to look for actual changes...